src/ViewLayer/DebugDisplay/BatteryPage/BatteryView/BatteryView.cpp

```diff
@@ -24,7 +24,7 @@ namespace
     const QString REQUESTED_FAN_ON = "background-color: rgb(64, 161, 191);";
     const QString FAN_OFF = "background-color: grey;";
 
-    const int MV_TO_V = 1000;
+    const float MV_TO_V = 1000;
```

Comment:
Did this need to be changed?

Reply:
Yes, an int * float in Qt is an int.

Reply:
> What changed on the UI file?

The Race mode dashboard had to be reorganized slightly since the conversion from mV to V made some decimal numbers that would get cut off as text.